src/topp/CometAdapter.cpp

```diff
@@ -752,6 +760,12 @@ class TOPPCometAdapter :
     // if "reindex" parameter is set to true will perform reindexing
     if (auto ret = reindex_(protein_identifications, peptide_identifications); ret != EXECUTION_OK) return ret;
 
+	// Parse ion mobility information if present
+	bool has_im = SpectrumMetaDataLookup::addMissingIMToPeptideIDs(peptide_identifications, exp);
```

Comment:
then: maybe instead of has_im something like all_ids_have_im
to express that everything identified was found and annotated with IM?